eslint.config.mjs

```diff
@@ -48,6 +48,7 @@ export default [
           allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing: false
         }
       ],
+      '@typescript-eslint/no-extraneous-class': 'off',
```

Comment:
Not sure there's a better way than a `static Ansi` class?

Reply:
You can also move those functions into the `tools/testers/helpers.ts` file or make an `ansi.ts` file in have them imported into ResultDisplays.ts

Reply:
Extracted to `ansi`.